easybib/libraries/deploy.rb

```diff
@@ -63,20 +63,19 @@ def allow_deploy(application, requested_application, requested_role = nil, node
     end
 
     def is_app_configured_for_stack(application, requested_application, requested_role, instance_roles)
-      case application
-      when requested_application
+      if application == requested_application
```

Comment:
I refactored this when I "read" the code in order to build a fixture for `node['opsworks']`.

cc @fh